network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/primary_heroguts.html

```diff
@@ -1,34 +1,41 @@
 {% load wagtailcore_tags wagtailimages_tags static %}
 
 <div id="hero">
-  <div class="banner">
-    {% with banner=page.specific.get_banner %}
-    {% if banner %}
-    <picture>
-      {% image banner fill-4032x1152 as image_xl %}
-      {% image banner fill-2400x686 as image_lg %}
-      {% image banner fill-1984x567 as image_md %}
-      {% image banner fill-1536x439 as image_sm %}
-      <source media="(min-width: 1200px)" srcset="{{ image_xl.url }}" alt="">
-      <source media="(min-width: 992px)" srcset="{{ image_lg.url }}" alt="">
-      <source media="(min-width: 768px)" srcset="{{ image_md.url }}" alt="">
-      <source media="(min-width: 576px)" srcset="{{ image_sm.url }}" alt="">
-      {# Fallback Image #}
-      {% image banner fill-1008x288 alt="" %}
-    </picture>
-    {% else %}
-    <img src="{% static "_images/banner-black-white-marble.jpg" %}" alt="">
+  <div class="banner{% if homepage %} homepage{% endif %}">
+    {% if homepage %}
+      <div class="content dark-theme">
```

Comment:
(Not sure if front end code needs review in this PR... is the code just for placeholder purpose only for now?)

If we want the layout to reflect more like the mockup in this PR - we should add the class `container` here.

Reply:
all front-end work in this PR is purely "stub" work, it's not meant to be even remotely correct, just there to make sure there is a place to pick up from. I'll take it out of this PR if we want to do this once all the backend changes have landed.